src/main.cpp

```diff
@@ -100,6 +100,9 @@ const QDBusArgument& operator>>(const QDBusArgument& arg, QImage& image);
 #include <QtPlugin>
 Q_IMPORT_PLUGIN(QSQLiteDriverPlugin)
 
+const char* legacy_portable_data_location = "data";
```

Comment:
These can be inside the anonymous namespace and should be named like `kLegacyPortableDataLocation`